pointcept/datasets/scannetpp.py

```diff
@@ -76,3 +76,28 @@ def get_data(self, idx):
         else:
             raise NotImplementedError
         return data_dict
+
+
+@DATASETS.register_module()
+class ScanNetPPTrainEvalSample(ScanNetPPDataset):
```

Comment:
The idea here is to not slow down training and just test on a small subset. Unlike semantic segmentation, nothing here is cached during the forward pass and and actual evaluation is done.